src/Graphics/Vty/Input/Events.hs

```diff
@@ -43,6 +43,7 @@ data Event
     -- row. Some terminals report only that a button was released
     -- without specifying which one; in that case, Nothing is provided.
     -- Otherwise Just the button released is included in the event.
+    -- Mouse wheel release events are not reported.
```

Comment:
Yeah, it's probably not crucial to inform users about the exact situation at this location.
I'll remove it!